devservices/config.yml

```diff
@@ -10,7 +10,7 @@ x-sentry-service-config:
       description: Service that provides fast aggregation and query capabilities on top of Clickhouse
       remote:
         repo_name: snuba
-        branch: master
+        branch: hubertdeng123/test-unrevert-topic-partition-counts-ci
```

Comment:
this change will be backed out before merge, but it helped us verify that tests would fail as in the prior unrevert, and that the Makefile change fixed them